src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -556,13 +556,17 @@ public virtual bool IsAnonymousType
         /// <summary>
         /// Is this a symbol for a Tuple
         /// </summary>
-        public virtual bool IsTupleType
-        {
-            get
-            {
-                return false;
-            }
-        }
+        public virtual bool IsTupleType => false;
+
+        /// <summary>
+        /// If this symbol represents a tuple type, get the types of the tuple's elements.
+        /// </summary>
+        public virtual ImmutableArray<ITypeSymbol> TupleElementTypes => default(ImmutableArray<ITypeSymbol>);
```

Comment:
From discussion with Aleksey, I will change the VB side to match this (return nothing) to align.